src/components/vuestic-components/va-divider/VaDivider.demo.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
I added some cases to demo a bit, as well as removed ambiguous ones.
Exclamation marks you can hover and check a tooltip for a hint.
![image](https://user-images.githubusercontent.com/5394573/74661318-c46c2a00-51a8-11ea-8bcb-c55b31f032ef.png)